OpenRA.Mods.Cnc/Traits/Render/WithRoof.cs

```diff
@@ -29,7 +29,7 @@ public class WithRoof
 		public WithRoof(Actor self, WithRoofInfo info)
 		{
 			var rs = self.Trait<RenderSprites>();
-			var roof = new Animation(self.World, rs.GetImage(self), () => self.Trait<IFacing>().Facing);
+			var roof = new Animation(self.World, rs.GetImage(self), RenderSprites.MakeFacingFunc(self));
```

Comment:
`MakeFacingFunc` is a wrapper around querying `IFacing`, with a fallback to 0 if it isn't defined.